src/openzaak/components/catalogi/admin/resultaattype.py

```diff
@@ -41,8 +41,21 @@ class ResultaatTypeAdmin(
         "toelichting",
     )
 
+    def get_extra_context(self, request, *args, **kwargs):
+        context = super().get_extra_context(request, *args, **kwargs)
+        self.zaaktype = context.get("zaaktype")
```

Comment:
I don't like setting random attributes during a method call, is there no better way?

Reply:
I have tried, but I don't think there is another way to access the request from the `get_zaaktype_procestype` zaaktype method, at least I couldn't find one